pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -28,89 +20,93 @@ fi
 
 # Install minimum dependencies
 if command -v apt-get >/dev/null 2>&1; then
-	DEBIAN_FRONTEND=noninteractive
-	export DEBIAN_FRONTEND
-	apt-get update
-	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ]; then
-		if ! command -v sshfs >/dev/null 2>&1; then
-			apt-get install -y sshfs
-		fi
+	pkgs=
+	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
```

Comment:
I wonder if `command -v $1 >/dev/null 2>&1` isn't used often enough to turn it into a function

```shell
function exists() { command -v $1 >/dev/null 2>&1; }
```

and then you can shorten conditions everywhere

```suggestion
	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! exists sshfs; then
```

Reply:
I think ``command -v` is short enough

Reply:
> I think ``command -v` is short enough

Well, it also hides the ugly (to me) `>/dev/null 2>&1` redirection, but yeah, it is just aesthetics.

Reply:
We can add that `function` (or a separate shell script under `/mnt/lima-cidata/util`), but we should do it for all `*.sh` at once (in another PR)